rcp/org.testeditor.rcp4.views.tcltestrun/src/org/testeditor/rcp4/views/tcltestrun/console/TestExecutionLogViewPart.xtend

```diff
@@ -25,7 +25,7 @@ class TestExecutionLogViewPart {
 
 	@PostConstruct
 	def void buildUI(Composite parent) {
-		browser = new Browser(parent, SWT.NORMAL)
+		browser = new Browser(parent, SWT.WEBKIT)
```

Comment:
Could this be a problem? On my machine (windows) this results in a 'org.eclipse.swt.SWTError: No more handles [Safari must be installed to use a SWT.WEBKIT-style Browser]' error.